druid/src/widget/svg.rs

```diff
@@ -83,12 +83,12 @@ impl<T: Data> Widget<T> for Svg<T> {
             bc.constrain(self.get_size())
         }
     }
-    fn paint(&mut self, paint_ctx: &mut PaintCtx, base_state: &BaseState, _data: &T, _env: &Env) {
+    fn paint(&mut self, paint_ctx: &mut PaintCtx, data: &T, env: &Env) {
```

Comment:
I think you still want the underscores here to prevent warnings, as the method doesn't use either data or env.

Reply:
Oh weird, I didn't get warnings for those. Fixed.

Reply:
Well, not surprising we didn't get warnings in CI, anyway :). Thanks!